accounts/migrations/0003_auto_20201213_1916.py

```diff
@@ -0,0 +1,55 @@
+# Generated by Django 3.1.4 on 2020-12-13 19:16
+
```

Comment:
You dont need multiple migration files as far as I know
@ifireball fix me if Im wrong....